packages/wallet-sdk/src/connection/WalletLinkConnection.ts

```diff
@@ -364,30 +344,35 @@ export class WalletLinkConnection {
   /**
    * Publish an event and emit event ID when successful
    * @param event event name
-   * @param unencryptedMessage unencrypted event message
+   * @param unencryptedData unencrypted event data
    * @param callWebhook whether the webhook should be invoked
    * @returns a Promise that emits event ID when successful
    */
-  public async publishEvent(event: string, unencryptedMessage: RelayMessage, callWebhook = false) {
+  public async publishEvent(
+    event: string,
+    unencryptedData: WalletLinkEventData,
```

Comment:
`RelayMessage` has been renamed.
it's actual payload that's encrypted and then eventually transported via WL websocket.